src/common/RadioButtons.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="flex items-baseline my-8">
+    <label :key="index" v-for="(icon, index) in icons">
+      <input class="opacity-0 absolute" name="size" type="radio" value="m" />
+      <component :is="icon"></component>
+    </label>
```

Comment:
lepszy byłby chyba radioButton jako oddzielny komponent do którego na slocie podajesz <component :is=".." />
https://v3.vuejs.org/guide/component-slots.html#slot-content